extensions/default/src/DicomWebDataSource/index.ts

```diff
@@ -243,6 +316,45 @@ function createDicomWebApi(dicomWebConfig: DicomWebConfig, servicesManager) {
        */
       getWadoDicomWebClient: () => wadoDicomWebClient,
 
+      /**
+       * Retrieves a thumbnail of an DICOM Instance.
+       *
+       * @param {Object} options
+       * @param {String} options.studyInstanceUID - Study Instance UID
+       * @param {String} options.seriesInstanceUID - Series Instance UID
+       * @param {String} options.sopInstanceUID - SOP Instance UID
+       * @param {MediaType[]} [options.mediaTypes] - Acceptable HTTP media types
+       * @param {Object} [options.queryParams] - HTTP query parameters
+       * @param {Request} [options.request] - Request Options - Request Options
+       * @returns {ArrayBuffer} Thumbnail
+       */
+      instanceThumbnail: async ({ studyInstanceUID, seriesInstanceUID, sopInstanceUID }) => {
```

Comment:
I don't see this being used anywhere.

Reply:
Let me ask Thomas - I don't think it ended up getting used.

Reply:
Removed it.